examples/data-sources/genesyscloud_conversations_messaging_integrations_open/data-source.tf

```diff
@@ -0,0 +1,3 @@
+data "genesyscloud_conversations_messaging_integrations_open" "integration_open_resource" {
```

Comment:
Just small suggestions to simplify this a bit 

`integration_open_resource` -> `example`

"data messaging open" -> "Example Open messaging integration name"